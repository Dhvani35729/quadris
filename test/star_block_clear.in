I
drop
I
right
right
right
right
right
right
right
drop
I
clockwise
right
right
right
right
drop
I
clockwise
right
right
right
right
right
right
drop
drop