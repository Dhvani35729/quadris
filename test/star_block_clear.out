Level:     0
Score:     0
Hi Score: 0
-----------
           
           
           
IIII       
           
           
           
           
           
           
           
           
           
           
           
           
           
           
-----------
Next:      
J  
JJJ


Level:     0
Score:     0
Hi Score: 0
-----------
           
           
           
IIII       
           
           
           
           
           
           
           
           
           
           
           
           
           
           
-----------
Next:      
J  
JJJ


Level:     0
Score:     0
Hi Score: 0
-----------
           
           
           
J          
JJJ        
           
           
           
           
           
           
           
           
           
           
           
           
IIII       
-----------
Next:      
OO 
OO 


Level:     0
Score:     0
Hi Score: 0
-----------
           
           
           
IIII       
           
           
           
           
           
           
           
           
           
           
           
           
           
IIII       
-----------
Next:      
OO 
OO 


Level:     0
Score:     0
Hi Score: 0
-----------
           
           
           
 IIII      
           
           
           
           
           
           
           
           
           
           
           
           
           
IIII       
-----------
Next:      
OO 
OO 


Level:     0
Score:     0
Hi Score: 0
-----------
           
           
           
  IIII     
           
           
           
           
           
           
           
           
           
           
           
           
           
IIII       
-----------
Next:      
OO 
OO 


Level:     0
Score:     0
Hi Score: 0
-----------
           
           
           
   IIII    
           
           
           
           
           
           
           
           
           
           
           
           
           
IIII       
-----------
Next:      
OO 
OO 


Level:     0
Score:     0
Hi Score: 0
-----------
           
           
           
    IIII   
           
           
           
           
           
           
           
           
           
           
           
           
           
IIII       
-----------
Next:      
OO 
OO 


Level:     0
Score:     0
Hi Score: 0
-----------
           
           
           
     IIII  
           
           
           
           
           
           
           
           
           
           
           
           
           
IIII       
-----------
Next:      
OO 
OO 


Level:     0
Score:     0
Hi Score: 0
-----------
           
           
           
      IIII 
           
           
           
           
           
           
           
           
           
           
           
           
           
IIII       
-----------
Next:      
OO 
OO 


Level:     0
Score:     0
Hi Score: 0
-----------
           
           
           
       IIII
           
           
           
           
           
           
           
           
           
           
           
           
           
IIII       
-----------
Next:      
OO 
OO 


Level:     0
Score:     0
Hi Score: 0
-----------
           
           
           
OO         
OO         
           
           
           
           
           
           
           
           
           
           
           
           
IIII   IIII
-----------
Next:      
 SS
SS 


Level:     0
Score:     0
Hi Score: 0
-----------
           
           
           
IIII       
           
           
           
           
           
           
           
           
           
           
           
           
           
IIII   IIII
-----------
Next:      
 SS
SS 


Level:     0
Score:     0
Hi Score: 0
-----------
I          
I          
I          
I          
           
           
           
           
           
           
           
           
           
           
           
           
           
IIII   IIII
-----------
Next:      
 SS
SS 


Level:     0
Score:     0
Hi Score: 0
-----------
 I         
 I         
 I         
 I         
           
           
           
           
           
           
           
           
           
           
           
           
           
IIII   IIII
-----------
Next:      
 SS
SS 


Level:     0
Score:     0
Hi Score: 0
-----------
  I        
  I        
  I        
  I        
           
           
           
           
           
           
           
           
           
           
           
           
           
IIII   IIII
-----------
Next:      
 SS
SS 


Level:     0
Score:     0
Hi Score: 0
-----------
   I       
   I       
   I       
   I       
           
           
           
           
           
           
           
           
           
           
           
           
           
IIII   IIII
-----------
Next:      
 SS
SS 


Level:     0
Score:     0
Hi Score: 0
-----------
    I      
    I      
    I      
    I      
           
           
           
           
           
           
           
           
           
           
           
           
           
IIII   IIII
-----------
Next:      
 SS
SS 


Level:     0
Score:     0
Hi Score: 0
-----------
           
           
           
 SS        
SS         
           
           
           
           
           
           
           
           
           
    I      
    I      
    I      
IIIII  IIII
-----------
Next:      
ZZ 
 ZZ


Level:     0
Score:     0
Hi Score: 0
-----------
           
           
           
IIII       
           
           
           
           
           
           
           
           
           
           
    I      
    I      
    I      
IIIII  IIII
-----------
Next:      
ZZ 
 ZZ


Level:     0
Score:     0
Hi Score: 0
-----------
I          
I          
I          
I          
           
           
           
           
           
           
           
           
           
           
    I      
    I      
    I      
IIIII  IIII
-----------
Next:      
ZZ 
 ZZ


Level:     0
Score:     0
Hi Score: 0
-----------
 I         
 I         
 I         
 I         
           
           
           
           
           
           
           
           
           
           
    I      
    I      
    I      
IIIII  IIII
-----------
Next:      
ZZ 
 ZZ


Level:     0
Score:     0
Hi Score: 0
-----------
  I        
  I        
  I        
  I        
           
           
           
           
           
           
           
           
           
           
    I      
    I      
    I      
IIIII  IIII
-----------
Next:      
ZZ 
 ZZ


Level:     0
Score:     0
Hi Score: 0
-----------
   I       
   I       
   I       
   I       
           
           
           
           
           
           
           
           
           
           
    I      
    I      
    I      
IIIII  IIII
-----------
Next:      
ZZ 
 ZZ


Level:     0
Score:     0
Hi Score: 0
-----------
    I      
    I      
    I      
    I      
           
           
           
           
           
           
           
           
           
           
    I      
    I      
    I      
IIIII  IIII
-----------
Next:      
ZZ 
 ZZ


Level:     0
Score:     0
Hi Score: 0
-----------
     I     
     I     
     I     
     I     
           
           
           
           
           
           
           
           
           
           
    I      
    I      
    I      
IIIII  IIII
-----------
Next:      
ZZ 
 ZZ


Level:     0
Score:     0
Hi Score: 0
-----------
      I    
      I    
      I    
      I    
           
           
           
           
           
           
           
           
           
           
    I      
    I      
    I      
IIIII  IIII
-----------
Next:      
ZZ 
 ZZ


Level:     0
Score:     0
Hi Score: 0
-----------
           
           
           
ZZ         
 ZZ        
           
           
           
           
           
           
           
           
           
    I I    
    I I    
    I I    
IIIII IIIII
-----------
Next:      
TTT
 T 


Level:     0
Score:     0
Hi Score: 0
-----------
           
           
           
TTT        
 T         
           
           
           
           
           
           
           
           
           
    I I    
ZZ  I I    
 ZZ I I    
IIIII IIIII
-----------
Next:      
  L
LLL